app/assess/data.py

```diff
@@ -342,10 +342,7 @@ def submit_score_and_justification(
     current_app.logger.info(
         f"Response from Assessment Store: '{response.json()}'."
     )
-    if response.status_code == 200:
-        return True
-    else:
-        return False
+    return response.ok
```

Comment:
Just curious. How is it different from `status_code` ?

Reply:
Nice doc string from the source library 😄

https://github.com/psf/requests/blob/aa4cc78627bcf8ff87b73422ec06748d833c7a15/requests/models.py#L755-L768